moto/autoscaling/responses.py

```diff
@@ -81,6 +81,7 @@ def create_auto_scaling_group(self):
             min_size=self._get_int_param("MinSize"),
             instance_id=self._get_param("InstanceId"),
             launch_config_name=self._get_param("LaunchConfigurationName"),
+            launch_template=self._get_dict_param("LaunchTemplate."),
```

Comment:
Is the `.` supposed to be there?

Reply:
Yup, without including the period it gets included in the key names within the dict.
https://github.com/spulec/moto/blob/385c78a996f07dd3d603acb2a2c9bdc799670fa4/moto/core/responses.py#L531-L544

Example in existing code:
https://github.com/spulec/moto/blob/e00af2f73cb7d27c3755f18b2161b9acbd8ca8aa/moto/ec2/responses/launch_templates.py#L101